lib/ruby/lib/alces/packager/repository.rb

```diff
@@ -125,12 +131,14 @@ def packages
 
       def update!
         if metadata.key?(:source)
-          case Alces.git.sync(repo_path, metadata[:source])
+          case r = Alces.git.sync(repo_path, metadata[:source])
```

Comment:
It's used [here](https://github.com/alces-software/clusterware/pull/168/files#diff-8fc2d801996e2afb524f8fffd7c86717R156); I've not changed this line but it was giving an error about `r` being unset (rather than the unrecognized response), so I assumed `r` was supposed to be set to the response here.